src/pages/_app.tsx

```diff
@@ -26,23 +25,23 @@ const NAVIGATION: Navigation = [
     icon: <DashboardIcon />,
   },
   {
-    segment: 'orders',
-    title: 'Orders',
-    icon: <ShoppingCartIcon />,
+    segment: 'listings',
+    title: 'Your Listing',
+    icon: <HomeIcon />,
   },
 ];
 
 export default function RootLayout({ children, Component, pageProps }: AppProps & { children: React.ReactNode }) {
-  const router = useRouter(); // Get current route
-  const [isRegisterRoute, setIsRegisterRoute] = React.useState(false);
-
-  React.useEffect(() => {
-    setIsRegisterRoute(router.pathname.startsWith('/register'));
-  }, [router.pathname]);
-
+  const router = useRouter();
   const [session, setSession] = React.useState<Session | null>(null);
+  const [isClient, setIsClient] = React.useState(false);
+  const [excludeLayout, setExcludeLayout] = React.useState(false);
 
+  // Detect if the current route should exclude the layout (e.g., /login, /register)
   React.useEffect(() => {
+    setIsClient(true);
+    setExcludeLayout(['/login', '/register'].includes(router.pathname));
+
     const token = Cookies.get('auth_token');
     if (token) {
       fetch('/api/me', {
```

Comment:
please change name from me to get user session. makes it easier to understand

Reply:
this is done
will push the change in 10 mins